sorc/ncep_post.fd/SURFCE.f

```diff
@@ -1220,6 +1221,229 @@ SUBROUTINE SURFCE
           endif
       ENDIF
 
+          IF ( IGET(996)>0 )THEN
+            if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(996))
+!$omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = LANDFRAC(i,jj)
+                enddo
+              enddo
+            endif
+          ENDIF
+
+          IF ( IGET(997)>0 )THEN
+            if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(997))
+!$omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = PAHI(i,jj)
+                enddo
+              enddo
+            endif
+          ENDIF
+
+          IF ( IGET(998)>0 )THEN
+            if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(998))
+!$omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = TWA(i,jj)
+                enddo
+              enddo
+            endif
+          ENDIF
+
+          IF ( IGET(999)>0 )THEN
+!$omp parallel do private(i,j)
+         DO J=JSTA,JEND
+           DO I=1,IM
+             GRID1(I,J) = TECAN(I,J)
+           ENDDO
+         ENDDO
+         ID(1:25) = 0
+         ITPREC     = NINT(TPREC)
+!mp
```

Comment:
@HelinWei-NOAA Please remove this comment line.

Reply:
removed